coding/javascript.md

```diff
@@ -23,88 +23,85 @@ On all publicly accessible websites, core content and functionality should still
 
 ## Element hooks
 
-We should target page element in Javascript using the `data-js` element attribute instead of using the class attribute. Classnames should be used for styling only.
+We should target page element in JavaScript using the `data-js` element attribute instead of using the `class` attribute. Classnames should be used for styling only.
 
 e.g. `<a data-js="index-link" href="/index">Index</a>`.
 
 ## Tooling
 
 ### Dependency management
 
-We use [yarn](https://yarnpkg.com/en) for dependency management.
+We use [Yarn](https://yarnpkg.com/en) for dependency management.
 
 ### Testing
 
-We use the [jest](https://jestjs.io) unittest framework.
+We use the [Jest](https://jestjs.io) unittest framework.
 
 ### Code style & formatting
 
 To ensure code formatting consistency across our codebase, we use [Prettier](https://github.com/prettier/prettier).
 
-JS should also be written using [AirBnB](https://github.com/airbnb/javascript) style. New projects can adhere to this by selecting 'AirBnB' during `eslint init`, and existing projects can find the relevant eslint config in [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb).
+JavaScript should also be written using [AirBnB](https://github.com/airbnb/javascript) style. New projects can adhere to this by selecting 'AirBnB' during `eslint init`, and existing projects can find the relevant ESLint config in [eslint-config-airbnb](https://www.npmjs.com/package/eslint-config-airbnb).
 
 ## React
 
 ### Starting a new project
 
-When starting a new react application, we recommend [create-react-app](https://github.com/facebook/create-react-app) for bootstrapping the project. Avoid [ejecting](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject) as long as feasible to make updating dependencies easier.
+When starting a new React application, we recommend [create-react-app](https://github.com/facebook/create-react-app) for bootstrapping the project. Avoid [ejecting](https://facebook.github.io/create-react-app/docs/available-scripts#npm-run-eject) as long as feasible to make updating dependencies easier.
 
 ### Preferred libraries
 
 All projects should generally use the following:
 
-* style framework - [vanilla](https://github.com/vanilla-framework/vanilla-framework)
-* component testing - [enzyme](https://github.com/airbnb/enzyme)
+* style framework - [Vanilla](https://github.com/vanilla-framework/vanilla-framework)
```

Comment:
What you've done here is great and much appreciated. But if you feel like going one step further - I think these list items should technically start with capital letters, since they follow a colon above. E.g.:

```suggestion
* Style framework - [Vanilla](https://github.com/vanilla-framework/vanilla-framework)
```